core/ov/include/libov/ov_ov.h

```diff
@@ -184,7 +184,7 @@ typedef OV_ENUM OV_VAR_PROPS;
 	#define OV_VL_MININT	INT32_MIN
 	#define OV_VL_MININT64	INT64_MIN
 #else
-	typedef	boot_t	OV_BOOL;		/* bool value */
+	typedef	bool_t	OV_BOOL;		/* bool value */
```

Comment:
Was this really a typo? I am confused :)

Reply:
I think it is a typo that was never hit...most "common" compilers somehow use stdint and thus the problem never occurs